homeassistant/components/overkiz/translations/en.json

```diff
@@ -1,20 +1,24 @@
 {
     "config": {
         "abort": {
-            "already_configured": "Device is already configured"
+            "already_configured": "Account is already configured"
```

Comment:
Is it correct that `strings.json` is not being updated?

Reply:
Yes. I didn't realize that this was not part of pre-commit, thus I forgot to build translations in previous PR. 

This PR now reflects the current strings.json and for follow up PR's, I will include both changes.